.azure/templates/build-config-user.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     inputs:
       pwsh: true
       filePath: scripts/prepare-machine.ps1
-      arguments: -Tls ${{ parameters.tls }} -ForBuild ${{ parameters.extraPrepareArgs }}
+      arguments: -Tls ${{ parameters.tls }} -ForBuild ${{ parameters.extraPrepareArgs }} -UseXdp
```

Comment:
Like I said in a separate comment, `prepare-machine.ps1` should just always install the XDP SDK when building Windows now.